gulpfile.js

```diff
@@ -208,7 +208,7 @@ function buildExtensions(options) {
 
   let extensionsToBuild = [];
   if (!!argv.extensions && !options.compileAll) {
-    extensionsToBuild = argv.extensions.split(',');
+    extensionsToBuild = getExtensionsFromArg(argv.extensions);
```

Comment:
As mentioned in a comment below, I think it's cleaner to use a separate arg for this. You'll need to document it in these spots:

https://github.com/ampproject/amphtml/blob/d4a3fd5c85a2762ac08a30758fb99a74ecfbd3fd/gulpfile.js#L1542-L1548
https://github.com/ampproject/amphtml/blob/d4a3fd5c85a2762ac08a30758fb99a74ecfbd3fd/gulpfile.js#L1553-L1559
This spot is missing `noextensions` and `extensions` too:
https://github.com/ampproject/amphtml/blob/d4a3fd5c85a2762ac08a30758fb99a74ecfbd3fd/gulpfile.js#L1560-L1568
https://github.com/ampproject/amphtml/blob/d4a3fd5c85a2762ac08a30758fb99a74ecfbd3fd/gulpfile.js#L1569-L1577

Finally, add a line to all the relevant spots in `contributing/DEVELOPING.md`